docs/code_examples/console_async.py

```diff
@@ -2,7 +2,6 @@
 import logging
 
 from aioconsole import ainput
-
```

Comment:
Why are all those blank lines removed?
If it is because of isort, flake8 or black, please make sure that you have the correct dev dependencies by running `pip install -e .[dev]`. The dev dependencies are pinned on a specific version to avoid problems like this.

Reply:
I've done exactly that, but I don't know what's wrong with my local setup. I'm having other errors also because of some failing tests from the `test_cli.py` file. Anyways, better solution for now is to revert those changes

Reply:
Are you sure you used the `-e` flag with pip? I know that if I don't use it, then the code coverage is skipping some lines (510-539) from `cli.py` for some reason...

Reply:
Ok I'll try with bash next time, I think zsh is incorrectly parsing the command. It should be it

Reply:
Yes, I believe with zsh you need to use quotes: `pip install -e ".[dev]"`